templates/donate.html

```diff
@@ -45,18 +44,29 @@ <h1>Donate Bitcoin</h>
         </div>
 
         <div id="paymentForm">
-            <center>
-                <form id="pay" action='/pay' style="margin:0;padding0;">
-                    <h2 style="margin:0;padding0;">Amount:
+            <form id="pay" action='/pay'>
+                <div style="display:block;text-align: center;">
+                    <h2 style="margin:0;">Amount:
                     <input id="amountenter" style="display:inline" size="4" type="float" name="amount" id="amount" placeholder="{{ params.currency }}" required>
                     </h2>
                     <br>
                     <input class="button button1" style="width:40%" type="submit" value="Donate">
-                </form>
-            </center>
+                </div>
+            </form>
+            <div id="paybutton"></div>
+        </div>
+        <br>
+        <div id="row">
+            <div id="left">
+                {% if params.node_info %}
+                <small><a id="about" href="/node/" target="_blank">Open a channel with me!</a></small>
```

Comment:
nit - maybe this way is more obvious?
```suggestion
                <small><a id="about" href="/node/" target="_blank">Open a Lightning channel with me!</a></small>
```